app/views/pair_requests/index.html.erb

```diff
@@ -1,42 +1,46 @@
 <%= link_to "New pair request", new_pair_request_path %>
-
 <h1 class="text-2xl font-bold mb-4 text-center">My Requests</h1>
-
-<div class="p-2">
-  <div class="table w-full border-2 border-black text-xs sm:text-sm md:text-base">
-    <div class="table-header-group">
-      <div class="table-row font-bold bg-neutral-400 divide-x-2 divide-black">
-        <div class="table-cell text-left p-1">Partner</div>
-        <div class="table-cell text-left p-1">When</div>
-        <div class="table-cell text-left p-1">Status</div>
-        <div class="table-cell text-left p-1">Actions</div>
-      </div>
+<%=  turbo_frame_tag "pair_requests" do %>
```

Comment:
Now that this container is a turbo frame, all actions/nav inside of this container are affected by that. If you try visiting any of the links and buttons on the items in the table, you'll see that things are broken. This is because links and buttons inside of a turbo frame expect to use the turbo format, and none of this stuff can handle that as written.

Because it's a completely different page, the links to the pair request show page can probably just use the `data-turbo-frame="_top"` attribute to trigger a full page load.

With buttons like 'Accept', 'Reject', 'Complete', you have a choice. You could just trigger a normal page load. Or you could use turbo to have it where the old buttons are replaced by whatever new thing should be rendered (ie. after a user completes a request, now it should have an 'edit feedback' link). You're free to think that's out of scope for this PR though. It is important to have all the buttons and links working though, whether you get fancy with it or not.